.github/scripts/upload-artifacts.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+s3_bucket="snark-artifacts"
+artifacts=$(ls packages/ | grep -v 'artifacts\|cli')
+
+cd packages
+git restore-mtime
+
+for artifact in $artifacts; do
+  aws s3 sync $artifact s3://$s3_bucket/$artifact --delete
+done
```

Comment:
as discussed on discord, this is great to deal with a single file version.

snark artifacts is about packaging the artifacts as npm packages, and then distributing them via CDN to allow consumers download them at runtime (and not bloat their deps).

So a requirement is that the bucket must serve all the versions that exists in the npm registry.

could we have something like
`/project/version/file`

e.g

`/poseidon/1.0.0/poseidon-1.zkey`
...

This is what I tried to implement in 
https://github.com/privacy-scaling-explorations/snark-artifacts/pull/90

where I read the package version from the package.json before copying to the bucket

Reply:
> as discussed on discord, this is great to deal with a single file version.
> 
> snark artifacts is about packaging the artifacts as npm packages, and then distributing them via CDN to allow consumers download them at runtime (and not bloat their deps).
> 
> So a requirement is that the bucket must serve **all** the versions that exist in the npm registry.
> 
> could we have something like `/project/version/file`
> 
> e.g
> 
> `/poseidon/1.0.0/poseidon-1.zkey` `/semaphore/4.0.0-beta.9/semaphore-2.wasm` etc ...
> 
> This is what I tried to implement in #90
> 
> where I read the package version from the package.json before copying to the bucket

Sounds good!
Applying a change in the aux script to upload the specific files to their respective versions.